daemon/daemon.go

```diff
@@ -100,30 +108,30 @@ func (c *Command) canAccess(r *http.Request, user *auth.UserState) bool {
 		isUser = true
 	} else if err != errNoID {
 		logger.Noticef("unexpected error when attempting to get UID: %s", err)
-		return false
+		return accessForbidden
 	} else if c.SnapOK {
-		return true
+		return accessOK
 	}
 
 	if r.Method == "GET" {
```

Comment:
This entire if block can be simplified to a single `if r.Method == "GET" && (...` for better readability IMHO, but not insisting on that.

Reply:
I'm not sure I understand this comment from @stolowski

Reply:
`if r.Method == "GET" && ((isUser && c.UserOK) || c.GuestOK)) { return accesOk }` - that's what I mean; or just merge the two inner conditionals and leave method check as is.

Reply:
I don't find that more readable tbh